admin/fleetcommander/utils.py

```diff
@@ -115,3 +116,15 @@ def merge_settings(a, b):
         result[domain] = [index[key] for key in index]
 
     return result
+
+
+def get_free_port():
```

Comment:
Soup.Server has a method called get_port that you can use right after the instance has started